supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -32,6 +32,23 @@ To view the history of changes from 2020 through 2022, see the link:https://gith
 [[year-2024]]
 == 2024
 
+[[july-2024]]
+=== July 2024
+
+.Glossary entries
+* *xref:composite-content-view[composite content view]*: Updated glossary entry to use lowercase spelling
+* *xref:content-view[content view]*: Updated glossary entry to use lowercase spelling
+* *xref:nmstate[Nmstate]*: Added glossary entry
+* *xref:on-premise[on-premise]*: Updated glossary entry to include Red{nbsp}Hat OpenShift Container Platform definition
+* *Product*: Removed glossary entry because Red{nbsp}Hat Satellite no longer capitalizes it
+* *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry because this term has been replaced by "Capsure Server"
```

Comment:
Apologies, I had a typo in my suggestion

```suggestion
* *Red{nbsp}Hat Network Proxy Server*: Removed glossary entry because this term has been replaced by "Capsule Server"
```